test/test_logger.rb

```diff
@@ -14,7 +14,8 @@ def task
 
     def setup
         super
-        task.overwrite_existing_files = true
+        task.overwrite_existing_files = false
+        task.auto_rename_existing_files = true
```

Comment:
Unless I'm missing something, this is the _only_ use case you are testing now.

Make a common behavior case and subclass it for the two possible configurations (three if you're feeling like writing tests for overwrite_existing_files).

Reply:
True. I was manually changing these values for debugging. I am gonna stick to default values during setup and write specific tests.